tests/integration/userTests.py

```diff
@@ -144,7 +144,7 @@ def test_status_change_bad_uid(self):
         badUserId = {"uid": -100, "status": "denied"}
         response = self.app.post_json("/v1/update_user/",
             badUserId, expect_errors=True, headers={"x-session-id":self.session_id})
-        self.check_response(response, StatusCode.CLIENT_ERROR, "No users with that uid")
+        self.check_response(response, StatusCode.INTERNAL_ERROR)
```

Comment:
I don't know, but I added it back in for update_user specifically (since that was the one testing for it)